ivadomed/training.py

```diff
@@ -45,6 +48,10 @@ def train(model_params, dataset_train, dataset_val, training_params, path_output
                                 training. This training state is saved everytime a new best model is saved in the log
                                 directory.
         debugging (bool): If True, extended verbosity and intermediate outputs.
+        rank (int): the rank of the training function as a process. Default value is set to -1, 
```

Comment:
We probably should move `rank` to default to `-1` and move to last (just like how we added world_size to maximize retro compatibility.

Reply:
I agree. The challenge is that when `mp.spawn` generates an process, the function is called as `fn(rank, *args)` (see [link](https://pytorch.org/docs/stable/multiprocessing.html?highlight=multiprocessing#spawning-subprocesses)). One workaround would be to use `mp.Process`, which I am working on now.